packages/roosterjs-plugin-picker/lib/PickerDataProvider.ts

```diff
@@ -16,14 +16,17 @@ export interface PickerPluginOptions {
     // Vertical (the default, when this is false), will call shiftHighlight with up (false) and down (true).
     // Horizontal (when this is true), will call shiftHighlight with left (false) and right (true).
     isHorizontal?: boolean;
+
+    // Option for letting the editor handle autocomplete on node insertion
+    handleAutoComplete?: boolean;
 }
 
 export interface PickerDataProvider {
     // Function called when the plugin is intialized to register two callbacks with the data provider and a reference to the Editor.
     // The first is called in order to "commit" a new element to the editor body that isn't handled automatically by the editor plugin.
     // The second sets the isSuggesting value for situations wherethe UX needs to manipulate the suggesting state that's otherwise plugin managed.
     onInitalize: (
-        commitMentionCallback: (nodeToAvoiddd: HTMLElement) => void,
+        insertNodeCallback: (nodeToAvoiddd: HTMLElement) => void,

```

Comment:
Is this parameter name typo'd?

Reply:
You mean `nodeToAvoiddd`? Oh yeah I wondered about that too, I will remove the extra d's 😄

Reply:
Don't see it fixed on latest.

Reply:
nvm github being dumb